domain/lib/src/usecases/myspace/my_space_view_state.dart

```diff
@@ -72,3 +72,21 @@ class PreparingUpload extends ViewState {
   @override
   List<Object> get props => [fileInfo];
 }
+
+class DownloadFileSuccess extends ViewState {
```

Comment:
this should not be called `DownloadFileSuccess` because it only returns `DownloadTaskId` to track download progress. I think `DownloadFileViewState` is better.